<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XfstkDldrGui</class>
 <widget class="QMainWindow" name="XfstkDldrGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XfstkDldrGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="UserInterfaceTabDock">
        <property name="tabPosition">
         <enum>QTabWidget::South</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionModify_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionModify_Settings">
   <property name="icon">
    <iconset resource="xfstkresources.qrc">
     <normaloff>:/menu/options/icons/preferences-system-4.png</normaloff>:/menu/options/icons/preferences-system-4.png</iconset>
   </property>
   <property name="text">
    <string>Modify Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="xfstkresources.qrc">
     <normaloff>:/menu/help/icons/xfstk.png</normaloff>:/menu/help/icons/xfstk.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="xfstkresources.qrc">
     <normaloff>:/menu/help/icons/help-browser-2.png</normaloff>:/menu/help/icons/help-browser-2.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xfstkresources.qrc"/>
 </resources>
 <connections/>
</ui>
